eox_hooks/tests/tutor/integration_test_tutor.py

```diff
@@ -18,3 +25,21 @@ def test_current_settings_code_imports(self):
         import eox_hooks.edxapp_wrapper.backends.courses_p_v1  # isort:skip
         import eox_hooks.edxapp_wrapper.backends.enrollments_l_v1  # isort:skip
         import eox_hooks.edxapp_wrapper.backends.models_l_v1  # isort:skip
+
+    def test_info_view(self):
+        """
+        Tests the info view endpoint in Tutor
+        """
+        info_view_url = f'{self.base_url}/eox-hooks/eox-info'
+
+        # Simulate a GET request to the info endpoint using the full URL
```

Comment:
I think inline comments are not necessary, the code is self-explanatory.

Reply:
You are right,thanks